migrator/README.md

```diff
@@ -0,0 +1,261 @@
+# Migrator Tool
+
+The library takes in E.164 phone numbers with their corresponding BCP-47 country code. If inputted numbers match any of the supported internal renumbering ‘recipes’, which are created from official country documentation, said numbers will undergo a process of migration to convert them into the most up to date, dialable format. A given “recipe” is an internal record which holds data needed to migrate a stale number into one that is valid and dialable. 
+
+## Capabilities
+
+```Single Number Migration``` - input a single E.164 phone number with its corresponding BCP-47 country code. If there is an available migration recipe, the number will be converted to the new format based on the given recipe. The resulting string of the migration will be outputted to the console.
+
+```File Migration``` - input a text file location with one E.164 number per line along with the BCP-47 country code that corresponds to the numbers in the text file. All numbers in the text file that match available recipes will be migrated and a copy of the text file will be created with the updated numbers. The file path to the newly created text file will be outputted to the console.
+
+```Migration Report``` - find all phone numbers in a given input that were converted with given migration recipes. The report will also show the phone numbers that were already seen as valid before migrations as well as identify any numbers that were migrated into invalid formats based on known valid phone number ranges for the specified country code.
+
+```Custom Recipe Migrations``` - specify custom recipe files to use for migrations instead of the internal maintained file. Custom recipe migrations will not check if converted numbers are valid based on country formats. 
+
+```Web Application``` - As an alternative to using the tool from the command line, [click here](https://phone-number-migrator.uc.r.appspot.com/) to access the web application version. 
+
+## Required Parameters
+
+```(-n, --number | -f, --file)``` - the phone number input for a given migration. The number argument must be a single E.164 phone number. The file argument must be a path to a text file containing one E.164 phone number per line. For file migrations, the original text file will not be overwritten and instead a new text file will be created containing the migrated version of each phone number or the original phone number when a migration did not take place. Note: all phone numbers entered for migration will be sanitized by removing any whitespace, hyphens or curved brackets. If the number after this process is still not in E.164 format, a migration will not be able to be performed on it.
```

Comment:
"The number argument must be a single E.164 phone number"
maybe provide an example or two here, one sanitized, one not?